### ISSUE TITLE
Convention for separating and rendering view partials

### ISSUE THREAD

**Issue by herenow** [apotonick/cells#418]
Hi,

How do we separate our cells into partials? I know we can just call render, but is there any convention? 

For example, `render partial: 'menu'` will expect an `_menu.erb` file.

From what I understood, the `render :partial` (`Cell::ViewModel::Partial`) is just a way for us to access our old rails partials/views, right?

Currently, I'm putting my partials in a partials folder, and rendering it.

```erb
<section class="account-nav">
  <%= render 'partials/messages' %>
  <%= render 'partials/my_profile' %>
  <%= render 'partials/account_menu' %>
</section>
```

Is this the way to go, how are you guys doing it :) ?

**In depth scenario:**

I have a cell called `AccountNavCell`, it has two public methods `for_landing` and `for_app_header`. One is rendered in our home page, and the other is used on the rest of the internal pages.

So basically, I have the following files:

**app/cells/account_nav_cell.rb**
```ruby
class AccountNavCell < Cell::ViewModel
  def for_landing
    render
  end

  def for_app_header
    render
  end
end
```

**app/cells/account_nav/for_landing.erb**
```erb
<section class="account-nav account-nav--shadowed-white">
  <%= render 'partials/messages' %>
  <%= render 'partials/my_profile' %>
  <%= render 'partials/account_menu' %>
</section>
```

**app/cells/account_nav/for_app_header.erb**
```erb
<section class="account-nav">
  <%= render 'partials/account_menu' %>
  <%= render 'partials/messages' %>
  <%= render 'partials/my_profile' %>
</section>
```

This two templates share the same content, but with a different layout and content organization.

Thank you for the great gem :)

**Comment by apotonick** [comment]
In Cells, the concept of "partials" doesn't exist anymore, everything is just a view. A cell always has at least one of them. If you want to share views across cells, you can use view inheritance, change the `prefixes` or simply encapsulate the shared ones into a cell itself and than call it from the other two.

Makes sense? https://gitter.im/trailblazer/chat

**Comment by herenow** [comment]
Makes sense :) Thanks :thumbsup:

**Comment by apotonick** [comment]
I might pick up this issue for our new Trailblazer [cookbook](http://pro.trailblazer.to). Thanks! :heart:
